e2etests/web/regular_integration_tests/lib/screenshot_support.dart

```diff
@@ -71,25 +70,21 @@ Future<void> runTestWithScreenshots(
   test.integrationDriver(
     driver: driver,
     onScreenshot: (String screenshotName, List<int> screenshotBytes) async {
-      if (browser == 'chrome') {
-        final Image screenshot = decodePng(screenshotBytes);
-        final String result = compareImage(
-          screenshot,
-          updateGoldens,
-          '$screenshotName-$browser.png',
-          PixelComparison.fuzzy,
-          diffRateFailure,
-          forIntegrationTests: true,
-          write: updateGoldens,
-        );
-        if (result == 'OK') {
-          return true;
-        } else {
-          io.stderr.writeln('ERROR: $result');
-          return false;
-        }
-      } else {
+      final Image screenshot = decodePng(screenshotBytes);
+      final String result = compareImage(
+        screenshot,
+        updateGoldens,
+        '$screenshotName-$browser.png',
+        PixelComparison.fuzzy,
+        diffRateFailure,
+        forIntegrationTests: true,
+        write: updateGoldens,
```

Comment:
I wonder why we pass updateGoldens twice

Reply:
This code is the same for integration tests and unit tests. 

This is an option that is useful for unit tests developers. In the test, one can also specify if it's only for `write` purposes. It can be added to the test request: https://github.com/flutter/engine/blob/master/lib/web_ui/dev/test_platform.dart#L165

you are right though, since we didn't add it to the e2e, it seems redundant for integration tests :)